### ISSUE TITLE
Add option to select video resolution when opening videos with external player

### ISSUE THREAD

**Issue by maninthemiddle01** [libre-tube/LibreTube#5968]
### Describe your suggested feature

Currently, when opening a video using an external player in LibreTube, the video resolution is not selectable. It would be highly beneficial to have the ability to choose the desired video resolution before launching the video in an external player.

**Feature Request:**
Please consider adding an option to select the video resolution when opening a video with an external player. This feature would enhance the user experience and provide more control over the video playback quality.

**Proposed Solution:**
1. When the user selects the option to open a video with an external player, display a resolution selection dialog or dropdown menu.
2. The resolution options should be populated based on the available resolutions for the specific video.
3. Allow the user to choose their preferred resolution from the available options.
4. Once the user selects a resolution and confirms, launch the external player with the chosen resolution.

**Benefits:**
- Improved user experience by providing control over the video quality.
- Flexibility to choose the optimal resolution based on network conditions or device capabilities.
- Consistency with the resolution selection option available within the LibreTube app.

Thank you for considering this feature request. :)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Ehab-d** [comment]
Please, do not write issues using ChatGPT. Also note that YouTube separates video from audio, starting from high quality 720p, so the external player option won't work or the video will start without audio.

**Comment by Bnyro** [comment]
We already pass an adaptive HLS stream that contains multiple resolutions to external players - if these players you use are good enough they allow you to set the resolution.

*Also, if you want to use ChatGPT, spam your friends and family, but don't waste the time of open source volunteers.*

**Comment by maninthemiddle01** [comment]
Hello, thank you for your answer and time to respond. I didn't want to spam you and I also want to support open source software, so if I find a bug or have a feature request.

I tried it out and neither with Nextplayer nor VLC I found an option to choose between different resolutions. Though I noticed, that the resolution changes with network speed. Or are there better players which support this?

I wrote the issue, because e.g. in NewPipe there is an option to choose between resolution before opening a stream with an external app.
